content/en/examples/pods/config/redis-pod.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
 spec:
   containers:
   - name: redis
-    image: kubernetes/redis:v1
+    image: k8s.gcr.io/redis:v1
```

Comment:
do we need a specific image or we can use `image: redis:5` like https://github.com/kubernetes/website/pull/14112/files#diff-7cdce5d48ef944b468c25599bab54d57R8 ?

Reply:
The "official" `redis` image doesn't work with those `volumeMounts` set up, but this specific image is apparently set up to look in those directories